src/ModelContextProtocol.AspNetCore/ModelContextProtocol.AspNetCore.csproj

```
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <IsPackable>true</IsPackable>
+    <PackageId>ModelContextProtocol.AspNetCore</PackageId>
+    <Description>.NET ASP.NET Core SDK for the Model Context Protocol (MCP)</Description>
```

Nit: I would consider this to still be part of the C# SDK rather than its own ASP.NET Core SDK.
```suggestion
    <Description>ASP.NET Core extensions for the C# Model Context Protocol (MCP) SDK.</Description>
```